src/nft/components/profile/list/ListPage.tsx

```diff
@@ -317,9 +294,7 @@ export const ListPage = () => {
             <EthValueWrapper totalEthListingValue={!!totalEthListingValue}>
               {totalEthListingValue > 0 ? formatEth(totalEthListingValue) : '-'} ETH
             </EthValueWrapper>
-            {!!totalEthListingValue && !!ethPriceInUSD && (
-              <UsdValue>{formatUsdPrice(totalEthListingValue * ethPriceInUSD)}</UsdValue>
-            )}
+            {!!totalEthListingValue && !!usdcValue && <UsdValue>{usdcAmount}</UsdValue>}
```

Comment:
you don't need to check for `totalEthListingValue` here if its undefined `usdcValue` will be too